examples/src/examples/misc/gizmos.mjs

```diff
@@ -482,22 +512,23 @@ async function example({ canvas, deviceType, data, glslangPath, twgslPath, scrip
     this.focus();
 
     // Change gizmo mode keybinds
-    const setType = (value) => {
+    const setType = (/** @type {string} */ value) => {
         data.set('gizmo.type', value);
 
         // call method from top context (same as controls)
+        // @ts-ignore
         window.top.setType(value);
     };
 
-    const keydown = (e) => {
+    const keydown = (/** @type {{ shiftKey: boolean; ctrlKey: boolean; }} */ e) => {
```

Comment:
```suggestion
    /** @param {KeyboardEvent} e */
    const keydown = (e) => {
```

(same for others - the beauty of JSDoc is that you don't make it harder to read the real JS code with intermingled comments like that)

Reply:
Have you actually tried this? I have a feeling this doesn't work and that inserting the type before the parameter is the only solution. Just a hunch...

Reply:
Yep: https://www.typescriptlang.org/play?ts=4.6.2&filetype=js#code/PQKhAIAEAcEMCdYFtwG8DSBTAngIwPYIAmAogG6YB2ALgL7ibgjABQAxvpQM7XgDWOIvgDulcAF5wACkwBKCQD40LcKvAdu+ADaYAdFvwBzGbrbV4WrNlkBuFrRZA

Reply:
The only caveat is you are required to add a description to the parameter